tetris.py

```diff
@@ -22,6 +23,8 @@
 
 block_letters = ["I", "L", "L_rev", "O", "E", "Z", "Z_rev"]
 
+game_status = Enum('game_status', 'in_progress, game_lost, game_won')
```

Comment:
Name this like a class, so that it will stand out, and won't be confused with local variables:

```suggestion
GameStatus = Enum('GameStatus', 'in_progress, game_lost, game_won')
```

Reply:
Renamed to ENUM_game_status